projects/is-editable-textbox/src/lib/is-editable-textbox.component.html

```diff
@@ -0,0 +1,15 @@
+<div *ngIf="edit" class="input-group input-group-text">
+  <input #textInput
+         type="text"
+         class="form-control is-input"
```

Comment:
do not use 'is-input' class (I've been using it in cron-editor, but it needs to be removed as well). Import `..../mixins` and apply appropriate mixins